src/exports/CBL.def

```diff
@@ -7,7 +7,8 @@ CBL_DumpInstances
 
 CBL_Log
 CBL_Log_s
-CBL_SetLogLevel
+
+CBLLog_SetConsoleLevel
```

Comment:
While you're at it, you should also add the rest of the `CBLLog` API -- see lines 29-37 of CBL.exp.

Reply:
Done